src/pg_to_evalscript/javascript_processes/array_create.js

```diff
@@ -0,0 +1,23 @@
+function array_create(arguments) {
+  const { data = [], repeat = 1 } = arguments;
+
+  if (!Array.isArray(data)) {
+    throw new Error("Argument `data` is not an array.");
+  }
+
+  if (!Number.isInteger(repeat)) {
+    throw new Error("Argument `repeat` is not an integer.");
+  }
+
+  if (repeat < 1) {
+    throw new Error("Argument `repeat` has a minimum value of 1.");
```

Comment:
Probably some better error message should be used here, but I can't think of anything at the moment. fftr